engine/runtime/src/test/java/org/enso/compiler/EnsoCompilerTest.java

```diff
@@ -364,6 +373,12 @@ public void testExportFrom() throws Exception {
     parseTest("from prj.Data.Foo export Bar, Baz");
   }
 
+  @Test
+  @Ignore // wrong order of exported symbols
```

Comment:
@kazcw, this is the test I had to disable because exported names are both on LHS as well as RHS...